fre/make/tests/compilation/test_fre_make_run_fremake.py

```diff
@@ -0,0 +1,13 @@
+''' test "fre make" calls '''
+
+from click.testing import CliRunner
+
+from fre import fre
+
+runner = CliRunner()
+
+## assumes you're running pytest from repo root dir
+def test_fre_make_run_fremake_null_model():
```

Comment:
I think if we want to test the importability of scripts and the functionality of this tool, it would be better to have this actually use the functions inside of `run_fremake`. So maybe instead of using `runner.invoke(...)`:

At the top:
```
from fre.make import runFremake 
```

Then testing the tool
```
runFremake.fremake_run(yamlfile,platform,target)
```
Where the `assert` could check for things like a checkout script, makefile, and compile script, and that they were successful

From the way @ilaflott described it, my understanding is that defining the runner is good for testing click usage / options which can be done in the `fre/tests/test_fre_make_cli.py` file

Reply:
yeah that is my general position- `CliRunner` based tests in `fre/tests`, `from fre.subtool import FOO` style tests in `fre/subtool/tests`

i'm also sort of waiting for someone to show me why that pattern maybe won't and/or can't last, if that is the case

Reply:
Oh yeah I think we talked about that! I should have those changes somewhere(hopefully), sorry got a little sidetracked by the image/conda debugging. I'll get the actual test updated soon.

Reply:
Should be fixed now. Right now it just has the run-fremake compile and checks for the resulting executable, I'll add a lot more tests to the file once this CI stuff is all merged in.